### ISSUE TITLE
1.59a — InvocationTargetException: null (Client + Server)

### ISSUE THREAD

**Issue by jobaq007** [NillerMedDild/Enigmatica2Expert#1020]
**Modpack version**
1.59a

**Can the issue be reproduced?**
Yes

**Logs**
(See below)

**Issue**
It crashes whenever it tries to launch, i tried repair profile since that usually works at the stage however not success this time.

If this is a feature request, feel free to delete the template.

**Comment by MaxNeedsSnacks** [comment]
**Please do not post the crash log directly as part of the issue; that makes it a lot harder to read and solve the problem. Please upload your log to pastebin or another comparable site next time.**

That being said, do you have OptiFine installed? If so, please remove it and see if the game starts then.

**Comment by jobaq007** [comment]
Thanks yeah ill do it next time. and no i dont have optifine installed.

**Comment by MaxNeedsSnacks** [comment]
Can you give me your launch options real quick?

**Comment by jobaq007** [comment]
where is that? lol. I tried reverting however the problem persist.

**Comment by jobaq007** [comment]
I went over the usual protocol, check and run repair profile, update it to lastest java, revert back. I dont think my world is corrupted since i have not made that far.

https://pastebin.com/Wq1vRjQ1

**Comment by jobaq007** [comment]
When the game starts

(X)Close launcher when game starts

( )Keep launcher open

( )Hide launcher and re-open when game closes

Launcher Method

 (X) Native Launcher

( )JAR Launcher

**Comment by MaxNeedsSnacks** [comment]
I can't figure out what the problem is here...

**Comment by jobaq007** [comment]
Ok. i will keep troubleshooting if not copy my lastest save, delete the whole thing and reinstall

**Comment by Tomatoh** [comment]
SMP worked out of the box
Server is where I developed the issue

Ok from what I did was like I normally do 

1)BACKUP BACKUP BACKUP
2) Delete config, mods, schematic, and script, directory
3) copy all files from the new zip to the directory 
4) Run the serverstart.bat file

5) Crash

Then to fix what I did

create anew directory with just the new download in it and run the serverstart.bat and chnage eula
it create a new world.. works..
copy over my world to it and it also worked.. 

something in old directory keeping new download from working with the rest of the old files -world

hope that helps

**Comment by MaxNeedsSnacks** [comment]
There might be an issue with duplicate or outdated coremods, can you try deleting any subdirectories inside the mods directory, so like... "1.12.2" for example?

**Comment by NillerMedDild** [comment]
I delete all the coremods that hide in subfolders, so it shouldn't be that. It looks somewhat like file corruption.

**Comment by MaxNeedsSnacks** [comment]
Ok so I’m assuming the Quark ASM is crashing the game... but why?

**Comment by MaxNeedsSnacks** [comment]
Think I found a solution here: https://github.com/NillerMedDild/Enigmatica2Expert/issues/1027#issuecomment-476879695

TL;DR — delete the mixin.out directory after updating... I think. Please do let me know whether that works!

**Comment by jobaq007** [comment]
Hi all,
I had to unzip my save. Delete the whole thing and install again. I tranfers my world and it worked fine.
Before the launcher was failing, and when it work my world didnt load because the update kept the old and new version of mekanism.
Thank you guys

**Comment by MaxNeedsSnacks** [comment]
Ah, thanks for the info then ^^ Glad to hear it worked out

Kinda weird that you had to reinstall the instance completely... still something I should try to figure out
